SPECS/qemu-kvm/qemu-kvm.spec

```diff
@@ -18,6 +18,8 @@ Patch4:      CVE-2019-20175.patch
 Patch5:      CVE-2020-13659.patch
 Patch6:      CVE-2020-16092.patch
 Patch7:      CVE-2020-15863.patch
+# CVE-2016-7161 was fixed in 2.7.0, but the CVE database was not updated.
+Patch8:      CVE-2016-7161.nopatch
 Vendor:         Microsoft Corporation
```

Comment:
I suggest moving Vendor & Distribution before the patches and aligning accordingly.

Reply:
Fixed